### ISSUE TITLE
Load balancer fills disk with access logs

### ISSUE THREAD

**Issue by sonicsnes** [rancher/rancher#7028]
**Rancher Version:**
1.2.0

lb-service-haproxy:v0.4.2 seems to store quite verbose access logs duplicated inside both /var/log/haproxy/traffic and /var/log/messages. I unknowingly filled up an entire 24GB linode with relatively high-traffic over the period of a couple weeks. Perhaps the logs should be rotated? Is there a way to turn off these new access logs in 1.2.0?

**Comment by deniseschannon** [comment]
This isa fixed in the new haproxy release (v0.4.5).

**Comment by sangeethah** [comment]
With rancher/lb-service-haproxy:v0.4.5 , access logs are not present in /var/log/haproxy/traffic and /var/log/messages anymore.

If user wants to enable logging , then it can be enabled using the following haproxy config:

```
global
    log 127.0.0.1:8514 local0
    log 127.0.0.1:8514 local1 notice

defaults
    log global
    option tcplog
```
